data-conversion/src/main.rs

```diff
@@ -23,6 +25,10 @@ fn main() {
         /// type to convert to
         #[argh(option)]
         totype: String,
+
+        /// optional exponent for fixed_to_binary -> default is -1
+        #[argh(option, default = "String::from(\"-1\")")]
+        exp: String,
```

Comment:
Could this be an `isize` or `i32` or whatever instead of a string? That is, Argh itself can parse the string CLI arguments into numbers for you.